docs/getting-started/work-with-azure-function/js-work-with-azure-function.md

```diff
@@ -143,12 +141,58 @@ Now save the code above as `subscribe.js` and run it using `node subscribe "<con
     }
     ```
 
-7.  Run the funcion.
+10. Run the funcion.
    
     ```bash
     func start
     ```
 
+## Setup subscriber
+
+In Azure Web PubSub you can connect to the service and subscribe to messages through WebSocket connections. WebSocket is a full-duplex communication channel so service can push messages to your client in real time. You can use any API/library that supports WebSocket to do so. Here is an example in node.js:
+
+1.  First install required dependencies:
+
+    ```bash
+    npm init -y
+    npm install --save ws
+    npm install --save axios
+    ```
+
+2.  Then use WebSocket API to connect to service
+
+    ```javascript
+    const WebSocket = require('ws');
+    const axios = require('axios');
+
+    async function main() {
+    let funcUrl = 'http://localhost:7071';
+    if (process.argv.length == 2) {
```

Comment:
maybe use an index.html as a subscriber? you can simply call fetch as this chatapp does  https://github.com/Azure/azure-webpubsub/blob/main/samples/javascript/chatapp/public/index.html#L11 it can be a subscriber when it does not have the "Send" button

Reply:
Yes, we can, but host a page require customer to install http-server. This way of client will reduce dependency for them to start-up and easier to run cause npm should be naturally used as a `node` user. 
For static page sample, maybe a K-line chart would be better. 😄

Reply:
How about using the function app as the webpage host, a similar approach here: https://github.com/Azure/azure-functions-signalrservice-extension/pull/127/files#diff-bd889452ece70b05fc3cafd2ab4d5ac055b394d0d586b42fd6a1d0ce4f1b1a09R20

As well as a js one here:  https://stackoverflow.com/a/54116169/687394